client-v2/README.md

```diff
@@ -60,20 +116,66 @@ And to fix any issues that are automatically resolvable by eslint run:
 yarn lint-fix
 ```
 
-## Flow
-We are using flow for typing in Fronts V2. In order to install types for new
-libraries added run:
+| Uses        | For         | Config |
+| ------------|------------- |--- |
+| [Prettier](https://github.com/prettier/prettier)      | Anti-bikeshed Auto syntax formatting | [.prettierrc](.prettierrc)|
+| [.editorconfig](https://editorconfig.org/)      | Standard Editor formatting  |[.editorconfig](.editorconfig)|
+| [TSLint](https://palantir.github.io/tslint/)      | Typescript Linting | [tslint](tslint.json)|
 
-```bash
-yarn update-types
-```
+## Typescript
+We are using Typescript for typing in Fronts V2. 
 
-If you would like to overwrite types that are previously installed to newer
-versions that have been updated in the project pass the `--overwrite` flag to
-the above command.
+## File Structure
 
-If you want to run flow to check for type errors, run `yarn flow`.
+```
+src
+├── actions
+├── bundles
+├── components
+│   ├── CAPI
+│   ├── FrontsCAPIInterface
+│   ├── FrontsEdit
+│   ├── clipboard
+│   ├── inputs
+│   ├── layout
+│   └── util
+├── constants
+├── containers
+├── fixtures
+├── fonts
+│   ├── headline
+│   └── text
+├── images
+│   └── icons
+├── lib
+│   └── dnd
+├── reducers
+├── selectors
+├── services
+├── shared
+│   ├── actions
+│   ├── bundles
+│   ├── components
+│   ├── constants
+│   ├── fixtures
+│   ├── images
+│   ├── reducers
+│   ├── selectors
+│   ├── types
+│   └── util
+├── types
+└── util
+    └── sharedStyles
+```
 
-To check for flow coverage, run `yarn flow-coverage`.
+- Components, Actions, Reducers and Selectors are top level.
+    - The [App component](src/components/App.tsx) is the application entry point
+    - All reducers are combined in the [Root Reducer](src/reducers/rootReducer.ts) as per standard convention
+- `bundles` **what is the concept of a bundle?**
+- `constants` store high-level application constants such as theme styles and image paths
+- `services` contains the modules for requests to APIs such as CAPI and FaciaAPI
+- `lib` contains modules designed to be reusable such as the Drag N' Drop (dnd) module
+- `shared` is a library written using React and Redux and Typescript for typing designed to make it easier to develop tools which need to provide the ability to curate collections of content. [Read more.](src/shared/.README)
+- `types` at the top level store types for ??
```

Comment:
This is the top level store for types that have no obvious home: e.g. `Action` which is a union of things that are split across a few files. I wonder whether some of those (perhaps `State` / `Store`) could be move to places like the root reducer etc? Any thoughts @Reettaphant , @jonathonherbert , @vlbee

Reply:
I really like having all types in the types folder, I find it confusing when we define types in other files but that's just something I'm used to.

Reply:
Ok I'm happy with that too - I guess we just need to be consistent either way for newcomers to the project. Potentially we can extract some types from places then?